src/meta/src/backup_restore/restore.rs

```diff
@@ -43,6 +43,15 @@ pub struct RestoreOpts {
     pub meta_store_type: MetaBackend,
     #[clap(long, default_value_t = String::from(""))]
     pub sql_endpoint: String,
+    /// Username of sql backend, required when meta backend set to MySQL or PostgreSQL.
+    #[clap(long, default_value = "")]
+    pub sql_username: String,
+    /// Password of sql backend, required when meta backend set to MySQL or PostgreSQL.
+    #[clap(long, default_value = "")]
+    pub sql_password: String,
```

Comment:
```suggestion
    pub sql_password: Secret<String>,
```

Does this need to be secret as well?

Reply:
I'm not sure. Do we need it? @zwang28